packages/relay/src/lib/config/hbarSpendingPlanConfigService.ts

```diff
@@ -0,0 +1,315 @@
+/*
```

Comment:
If this is also a service, should we consider moving it into the `services/` folder? Perhaps something like `services/config/hbarSpendService.ts`?

Reply:
I thought about this as well, but it seems like it would be a better option to have all kinds of configurations stored in only one place - inside the `lib/config` directory. Otherwise configurations will be all scattered around in the folders and will be harder to track.